misc/src/main/java/com/example/snippets/backgroundwork/WakeLockSnippetsJava.java

```diff
@@ -18,7 +19,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         PowerManager powerManager = (PowerManager) getSystemService(POWER_SERVICE);
         PowerManager.WakeLock wakeLock =
                 powerManager.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "MyClassName::MyWakelockTag");
-        wakeLock.acquire();
+        wakeLock.acquire(wlTimeout);
```

Comment:
suggest using `WAKELOCK_TIMEOUT` which is evident that this is a const defined in the class